recipes/glue-geospatial/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.1" %}
+
+package:
+  name: glue-geospatial
+  version: {{version}}
+
+source:
+  fn: glue-geospatial-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/g/glue-geospatial/glue-geospatial-{{version}}.tar.gz
+  sha256: 95e33a0b317f3df406acffd808112f852b04687b0b1062467e044e99634a30dc
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Can you change this to the new pip variant as in the example

Reply:
Fixed